rhscl-postgresql-apb/README.md

```diff
@@ -0,0 +1,25 @@
+rhscl-postgresql-apb
+======================
+
+An apb for deploying [PostgreSQL](https://www.postgresql.org).
+
+## What it does
+* Deploys 1 postgresql container.
+
+## Requirements
+
+## Parameters
+* NAMESPACE: Optional, default 'postgresql-demo-apb', Namespace to deploy the cluster in.
+* POSTGRESQL_DATABASE, default 'admin', Postgresql database name.
+* POSTGRESQL_PASSWORD, default is a randomly generated string, Postgresql database password.
```

Comment:
woops, reply went to totally wrong comment:

hmm, with the env-file the file would have to exist on the container though, so you could only do that if you mounted the file into the container, right?